documentation/modules/retrieving-vmware-mor.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="retrieving-vmware-mor_{context}"]
+= Retrieving a VMware vSphere MOR
+
+When you migrate VMs with a VMware vSphere source provider using {project-first} from the CLI, you need to know the managed object reference (MOR) of certain entities in vSphere, such as datastores, networks, and VMs.
+
+You can retrieve the MOR of one or more vSphere entities from the Inventory service. You can then use each MOR as a reference for retrieving the MOR of another entity.
+
+.Procedure
+
+. Retrieve the routes for the project:
++
+[source,terminal]
+----
+oc get route -n openshift-mtv
+----
+
+. Retrieve the `Inventory` service route:
++
+[source,terminal,subs="attributes+"]
+----
+$ {oc} get route <inventory_service> -n {namespace}
+----
+
+. Retrieve the access token:
++
+[source,terminal]
+----
+$ TOKEN=$(oc whoami -t)
+----
+
+. Retrieve the `MOR` of a VMware vSphere provider:
```

Comment:
Why the backticks on this MOR but not earlier in the documentation?

Please be consistent with your formatting